coresimd/src/x86/i586/avx.rs

```diff
@@ -96,6 +96,7 @@ pub unsafe fn _mm256_or_ps(a: __m256, b: __m256) -> __m256 {
 #[inline]
 #[target_feature(enable = "avx")]
 #[cfg_attr(test, assert_instr(vshufpd, imm8 = 0x1))]
+#[rustc_args_required_const(2)]
```

Comment:
It looks like supporting attributes attached directly to function parameters would help with cases like this.
*\*Adds it to the list of things to implement some time\**

Reply:
Indeed!